dw-input.js

```diff
@@ -708,6 +708,7 @@ export class DwInput extends DwFormElement(LitElement) {
    */
   _toggleType() {
     this._type = this._type === 'text' ? 'password' : 'text';
+    this.focus();
```

Comment:
@nirmalbaldaniya-dream we should ideally use timeout to change the focus. You missed it here!